src/Compilers/CSharp/Portable/Lowering/DiagnosticsPass_ExpressionTrees.cs

```diff
@@ -17,10 +17,13 @@ internal sealed partial class DiagnosticsPass
         private readonly DiagnosticBag _diagnostics;
         private readonly CSharpCompilation _compilation;
         private bool _inExpressionLambda;
-        private LocalFunctionSymbol _staticLocalFunction;
         private bool _reportedUnsafe;
         private readonly MethodSymbol _containingSymbol;
 
+        // Containg static local function or static local anonymous function (anonymous method,
+        // lambda, etc.)
```

Comment:
>etc. [](start = 19, length = 4)

Are there more? Perhaps the comment could be: "Containing static local function, static lambda, or static anonymous method."

Reply:
will do.